tests/impls/rank_relevancy/test_random_sampling.py

```diff
@@ -0,0 +1,68 @@
+import numpy
+from typing import Dict, List, Sequence, Iterable
+
+from smqtk_relevancy.interfaces.rank_relevancy import RankRelevancy
+from smqtk_relevancy.impls.rank_relevancy.random_sampling import (
+    RankRelevancyWithRandomFeedback,
+)
+from smqtk_descriptors import DescriptorElement
+
+
+def test_is_usable() -> None:
+    assert RankRelevancyWithRandomFeedback.is_usable()
+
+
+class DummyRankRelevancy(RankRelevancy):
+    def get_config(self) -> Dict:
+        return {}
+
+    def rank(self, pos: Iterable[DescriptorElement], neg: Iterable[DescriptorElement], pool: Sequence) -> List:
+        return [v[0] for v in pool]
+
+
+def make_random_ranker(n: int, seed: int = 0) -> "RankRelevancyWithRandomFeedback":
```

Comment:
Return type doesn't have to be in quotes, but this is fine.

Reply:
Removed quotes, thanks.